rpc_jobs/cloud_image_build.yml

```diff
@@ -0,0 +1,59 @@
+- job:
+    name: Cloud-Image-Build
+    project-type: workflow
+    concurrent: false
+    properties:
+      - build-discarder:
+          num-to-keep: 30
+    parameters:
+      - instance_params
+      - rpc_gating_params
+    dsl: |
+      library "rpc-gating@${RPC_GATING_BRANCH}"
+
+      // These are defined statically instead of parameters
+      // to ensure that the job is always executed the same
+      // way.
+      env.FLAVOR = "performance1-1"
+      env.STAGES = "Allocate Resources, Save Slave, Cleanup"
+      image_list = [
+        [ src: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)",
```

Comment:
This means the source image will always be able to get built, but it also means our target is shifting, because everytime we'll build based on that, we'll have a different image id. 

So if we expect reproducibility, this is kinda good but not great.

Because Rackspace doesn't hold the "first release" image of each distro, should we either ask rackspace to have a "first release" image of each distro (and use this image id as source?) OR directly save the original image ID somewhere(to be able to track how this image was built?).
2) Directly do

Reply:
We don't expect reproducibility. When this job is activated, we want a new image built at the time based on the current image available.

Reply:
Good, that's exactly what I hoped and understood. I like evidence!